nni/nas/oneshot/pytorch/supermodule/base.py

```diff
@@ -104,3 +126,21 @@ def mutate(cls, module: nn.Module, name: str, memo: dict[str, Any], mutate_kwarg
             See :class:`BaseOneShotLightningModule <nni.retiarii.oneshot.pytorch.base_lightning.BaseOneShotLightningModule>` for details.
         """
         raise NotImplementedError()
+
+    def _slice_params_mapping(self):
+        return {}
```

Comment:
Looks like `_slice_params_mapping` is only used in MixedOperation's subclasses. You can put that in MixedOperation base-class.